keyboards/cutie_club/giant_macro_pad/chconf.h

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2021 Cutie Club
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define CH_CFG_ST_FREQUENCY 10000
+
+#define CH_CFG_OPTIMIZE_SPEED FALSE
+
+#define CH_CFG_USE_REGISTRY TRUE
+
+#define CH_CFG_USE_WAITEXIT TRUE
+
+#define CH_CFG_USE_CONDVARS TRUE
+
+#define CH_CFG_USE_CONDVARS_TIMEOUT FALSE
+
+#define CH_CFG_USE_MESSAGES TRUE
+
+#define CH_CFG_USE_MAILBOXES TRUE
```

Comment:
Can you explain why you need these? Is this file needed?

Reply:
Many apologies, my mistake, I do not think I need this file at all 😄 

I will delete them and mark this conversation as resolved

Reply:
Presuming ST_Frequency etc is now a QMK default based on MCU?

Would I need to specify a generic BOARD or is this handled by the build system?

Reply:
Defaults live under `platforms/chibios/common/configs` and `platforms/chibios/GENERIC_STM32_F072XB/configs`. What you'd done is override the defaults, asking for extra things from ChibiOS... which you weren't using.

Reply:
Main intention is that the defaults are present so that under most circumstances designers don't need to modify anything apart from the `MCU = xxxxxx` line in the rules.mk file.